src/components/DataTableDirective.js

```diff
@@ -53,7 +53,7 @@ export default function DataTableDirective($window, $timeout, $parse) {
                    on-page="dt.onBodyPage(offset, size)"
                    on-tree-toggle="dt.onTreeToggled(row, cell)">
            </dt-body>
-          <dt-footer ng-if="dt.options.footerHeight"
+          <dt-footer ng-if="dt.options.footerHeight || dt.options.paging.internal"
```

Comment:
As long as we're adding an `internal` flag, it should work without expecting the `footerHeight` option to be set too...